packages/core/src/App/Constants/routes-config.js

```diff
@@ -146,6 +146,11 @@ const getModules = ({ is_appstore }) => {
                             component: Account,
                             getTitle: () => localize('Proof of address'),
                         },
+                        {
+                            path: routes.proof_of_ownership,
+                            component: Account,
```

Comment:
is this correct? the component should be `Account`?

Reply:
Yes @ali-hosseini-fs, It should be `Account` as this is also used by the other parts such as `POI` and `POA` 
<img width="791" alt="routes" src="https://user-images.githubusercontent.com/104334373/201817892-7412023e-126f-44fa-952a-4e704a7df17e.png">

Reply:
got it 
thanks